src/nft/utils/timeSince.ts

```diff
@@ -27,19 +29,13 @@ const MINUTE = 1000 * 60
 const HOUR = MINUTE * 60
 const DAY = 24 * HOUR
 
-export const timeLeft = (targetDate: Date, mostSignificant = false): string => {
+export const timeLeft = (targetDate: Date): string => {
   const countDown = new Date(targetDate).getTime() - new Date().getTime()
   const days = Math.floor(countDown / DAY)
   const hours = Math.floor((countDown % DAY) / HOUR)
   const minutes = Math.floor((countDown % HOUR) / MINUTE)
 
-  if (mostSignificant) {
-    if (days > 0) return `${roundAndPluralize(days, 'day')}`
-    if (hours > 0) return `${roundAndPluralize(hours, 'hour')}`
-    if (minutes > 0) return `${roundAndPluralize(minutes, 'minute')}`
-  }
-
-  return `${days > 0 ? roundAndPluralize(days, 'day') : ''} ${hours > 0 ? roundAndPluralize(hours, 'hour') : ''} ${
-    minutes > 0 ? roundAndPluralize(minutes, 'minute') : ''
+  return `${days > 0 ? roundAndPluralize(days, t`day`) : ''} ${hours > 0 ? roundAndPluralize(hours, t`hour`) : ''} ${
+    minutes > 0 ? roundAndPluralize(minutes, t`minute`) : ''
```

Comment:
this translation and helper function won't work as it would just add `s` for pluralizations in other languages. There is a `Plural` component from Lingui

Reply:
slightly harder than that would have to reformat this file to a tsx to return a reactnode. Since its no longer being used I think its out of scope + its only used in details that is going to be refactored anyways